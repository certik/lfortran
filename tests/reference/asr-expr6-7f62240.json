{
    "basename": "asr-expr6-7f62240",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/expr6.f90",
    "infile_hash": "722af2f789e614d9acf6975aae8260e4949a360066d5aa53a3f05944",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-expr6-7f62240.stdout",
    "stdout_hash": "11e898294766270a65b3c5c778ddd8f7233de36af58d7285a2ff1eb4",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}