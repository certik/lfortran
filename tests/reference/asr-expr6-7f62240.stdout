(TranslationUnit (SymbolTable 1 {}) [(IntegerBinOp (IntegerConstant 5 (Integer 4 [])) Add (IntegerConstant 3 (Integer 4 [])) (Integer 4 []) (IntegerConstant 8 (Integer 4 []))) (RealConstant   5.29999999999999982e+00 (Real 4 []))])
